xml/hwbp_registers.xml

```diff
@@ -741,15 +745,23 @@
             </value>
 
             <value v="1" name="before">
-            The trigger fired just before the instruction that triggered it was
-            retired, but after all preceding instructions are retired.
+            The trigger fired before the instruction that matched it was
+            retired, but after all preceding instructions are retired. This
+            explicitly allows for instructions partially executing.
```

Comment:
"for instructions to be partially executed."

Reply:
Done.